modules/virt-creating-infiniband-interface-on-nodes.adoc

```diff
@@ -53,16 +54,30 @@ spec:
       type: infiniband <5>
 # ...
 ----
-<1> `datagram` is the default mode for an IPoIB interface, and this mode improves optimizes performance and latency. `connected` mode is a supported mode but consider only using this mode when you need to adjust the maximum transmission unit (MTU) value to improve node connectivity with surrounding network devices.  
```

Comment:
The rest of the config still references these numbered notes. If we're going to change the style we should update all of them.